hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/FinalizeRecordHandler.java

```diff
@@ -183,7 +183,7 @@ private void logHbarFinalizationFailInvalid(
                         .map(accountId -> String.format(
                                 "\tOriginal : %s%n\tModified : %s",
                                 accountStore.getOriginalValue(accountId), accountStore.get(accountId)))
-                        .collect(Collectors.joining("%n")));
+                        .collect(Collectors.joining("\n")));
```

Comment:
While the change makes sense, I hope nobody parses the old log message today. If someone does, then they need changing too.

Reply:
We'll use "scream test" approach.